extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -940,10 +952,18 @@ export class AmpA4A extends AMP.BaseElement {
             dev().error(TAG, this.element.getAttribute('type'),
                 'Error executing onAmpCreativeRender', err);
           })();
+          // Capture timing info for friendly iframe load completion.
+          getTimingDataAsync(friendlyIframeEmbed.win,
+              'navigationStart', 'loadEventEnd').then(delta => {
+                this.emitLifecycleEvent('renderFriendlyEnd', {
```

Comment:
This will cause renderFriendlyEnd's definition to change from after DOM attachment to load... just wanted to explicitly point that out